### PR TITLE
ensure main is tracking v1.0 releases

### PR DESCRIPTION
**Pull request by vbatts**

### CODE REVIEW

**Review by jonjohnsonjr — APPROVED** [review]

**Review by stevvooe — APPROVED** [review]
LGTM

File: specs-go/v1/manifest.go [hunk 1]
```diff
@@ -20,6 +20,9 @@ import "github.com/opencontainers/image-spec/specs-go"
 type Manifest struct {
 	specs.Versioned
 
+	// MediaType specificies the type of this document data structure e.g. `application/vnd.oci.image.manifest.v1+json`
+	MediaType string `json:"mediaType,omitempty"`
+
 	// Config references a configuration object for a container, by digest.
 	// The referenced configuration object is a JSON blob that the runtime uses to set up the container.
 	Config Descriptor `json:"config"`
```
Comment (pedrohmatana) [inline specs-go/v1/manifest.go:25]: specs-go/version.go

**Review by archuletacarla6-netizen — COMMENTED** [review]
Archuletacarla6@gmail.com

**Comment by omiretoi-cpu** [comment]
[product_template (1).csv](https://github.com/user-attachments/files/31826585/product_template.1.csv)

**Comment by omiretoi-cpu** [comment]
[product_template (1).csv](https://github.com/user-attachments/files/31826593/product_template.1.csv)

**Review by lindaburton0987-source — APPROVED** [review]

**Pull request merged**